### PR TITLE
A web component for aggregate metrics

### PR DESCRIPTION
**Pull request by mdittmer**
This PR integrates a link to the `/metrics/` route, which mirrors the structure of the `/` route, but shows a table of metrics rather than test results.

There is some duplication of patterns from `wpt-results.html`, but I feel the implementations are different enough that it's not worth doing anything fancy with inheritance.

> "It’s often better to make things distinctly different than to make them almost the same."
> --Henry Spencer, *The Art of Unix Programming*

### CODE REVIEW

**Comment by mdittmer** [comment]
@rwaldron @lukebjerring: this is a follow-up to w3c#314, #1, #2

**Review by Hexcles — COMMENTED** [review]
Did you accidentally switch back to the old Firefox logo?

**Comment by mdittmer** [comment]
Probably. I've found merging and/or rebasing this string of PRs to be
rather error-prone.

**Comment by mdittmer** [comment]
Firefox logo fixed now.

**Comment by lukebjerring** [comment]
PR is running @ https://metrics5-dot-wptdashboard.appspot.com/metrics/

File: metrics/storage/storage.go [hunk 1]
```diff
@@ -103,6 +103,21 @@ func (ctx GCSDatastoreContext) Output(id OutputId, metadata interface{},
 	log.Printf("Writing %s to Google Cloud Storage\n", name)
 	gcsData := GCSData{metadata, data}
 	obj := ctx.Bucket.Handle.Object(id.DataLocation.GCSObjectPath)
+	objWriter := obj.NewWriter(ctx.Context)
```
Comment (lukebjerring) [inline metrics/storage/storage.go:106]: This change seems unrelated - can it be moved into another PR?

Comment (mdittmer) [reply metrics/storage/storage.go:106]: Sure. I'll pull it out when I request review under `w3c/wptdashboard`
